gazebo_plugins/include/gazebo_plugins/gazebo_ros_ackermann_drive.hpp

```diff
@@ -39,6 +39,11 @@ class GazeboRosAckermannDrivePrivate;
 
       <update_rate>100.0</update_rate>
 
+      <!-- kinematics -->
+      <wheel_radius>0.3</front_left_joint>
+      <wheel_base>2.0</wheel_base>
+      <track_width>1.2</track_width>
```

Comment:
Could we please include a comment to indicate that this means "wheel_seperation" in ackermann language? Should help people to understand what really is just from reading.

Reply:
Done [7410afe](https://github.com/ros-simulation/gazebo_ros_pkgs/pull/1427/commits/7410afe37274be91661275230afbfe1d6dd630cb)